.github/workflows/ci.yml

```diff
@@ -32,73 +32,191 @@ jobs:
 
   injection-tests-java:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     strategy:
       matrix:
-        lib-injection-connection: ['uds', 'network']
+        lib-injection-connection: [ 'network','uds']
         lib-injection-use-admission-controller: ['', 'use-admission-controller']
+        weblog-variant: [ 'dd-lib-java-init-test-app']
```

Comment:
don't think the `weblog-variant` name makes sense. `weblog` has no meaning here. Maybe should be `testapp-variant`?